internal/install/execution/go_task_recipe_executor.go

```diff
@@ -186,37 +186,43 @@ func varsFromRecipe(r types.Recipe) (types.RecipeVars, error) {
 	return vars, nil
 }
 
-func varsFromInput(inputVars []recipes.VariableConfig) (types.RecipeVars, error) {
+func varsFromInput(inputVars []recipes.VariableConfig, assumeYes bool) (types.RecipeVars, error) {
 	vars := make(types.RecipeVars)
 
 	for _, envConfig := range inputVars {
 		envValue := os.Getenv(envConfig.Name)
-		if envValue == "" {
-			log.Debugf("required env var %s not found", envConfig.Name)
-			msg := fmt.Sprintf("value for %s required", envConfig.Name)
-
-			if envConfig.Prompt != "" {
-				msg = fmt.Sprintf("%s: %s", envConfig.Name, envConfig.Prompt)
-			}
 
-			prompt := promptui.Prompt{
-				Label: msg,
-			}
-
-			if envConfig.Secret {
-				prompt.HideEntered = true
-			}
-
-			if envConfig.Default != "" {
-				prompt.Default = envConfig.Default
-			}
-
-			result, err := prompt.Run()
-			if err != nil {
-				return types.RecipeVars{}, fmt.Errorf("prompt failed: %s", err)
+		if envValue == "" {
+			if assumeYes {
+				log.Debugf("required env var %s not found, using default value", envConfig.Name)
```

Comment:
@zlesnr Is there any issue when the assumeYes is used but a recipe requires an input to be entered and no default is provided?
I believe in this case some sort of error should raised (maybe this is done somewhere else), or alternatively the question should be prompted (perhaps bad given the input said not to?).

Reply:
Good callout.  I'll draft a workaround.

Reply:
I've updated the code to check for empty default value and return an error if none were found.  How's that?